README.md

```diff
@@ -77,4 +77,3 @@ hooks are:
 [pre-commit]: https://pre-commit.com/
 [web-readme]: ./web/README.md
 [lib-readme]: ./rust/stackable-cockpit/README.md
-[xtasks]: ./xtask/src/main.rs
```

Comment:
question: I assume this was unused and therefore removed?

Reply:
Exactly, pre-commit complained about it being unused